src/commands/importFunction/importFunctionApp.ts

```diff
@@ -0,0 +1,405 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ApiContract, BackendContract, BackendCredentialsContract, OperationCollection, OperationContract, PropertyContract } from "azure-arm-apimanagement/lib/models";
+import { WebSiteManagementClient } from "azure-arm-website";
+import { Site, WebAppCollection } from "azure-arm-website/lib/models";
+import { ServiceClientCredentials, WebResource } from "ms-rest";
+import * as request from 'request-promise';
+import { ProgressLocation, window } from "vscode";
+import { appendExtensionUserAgent, createAzureClient } from "vscode-azureextensionui";
+import xml = require("xml");
+import * as Constants from "../../constants";
+import { ApisTreeItem } from "../../explorer/ApisTreeItem";
+import { ApiTreeItem } from "../../explorer/ApiTreeItem";
+import { ServiceTreeItem } from "../../explorer/ServiceTreeItem";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { IAzureClientInfo } from "../../models/AzureClientInfo";
+import { IFunctionKeys } from "../../models/Function";
+import { apiUtils } from "../../utils/apiUtils";
+import { nRequest, requestUtilWithCredentials, sendRequest } from "../../utils/requestUtil";
+import { signRequest } from "../../utils/signRequest";
+import { Utils } from "../../utils/Utils";
+import { IFunctionContract } from "./IFunctionContract";
+
+export async function importFunctionAppToApi(node?: ApiTreeItem): Promise<void> {
+    if (!node) {
+        node = <ApiTreeItem>await ext.tree.showTreeItemPicker(ApiTreeItem.contextValue);
+    }
+
+    ext.outputChannel.show();
+    ext.outputChannel.appendLine(localize("importFunctionApp", `Import function app to api ${node.root.apiName} started...`));
+    ext.outputChannel.appendLine(localize("importFunctionApp", "Create website client..."));
```

Comment:
Dont need this comment. Avoid giving implementation details rather give progressive information.